src/app/views/home/home.component.scss

```diff
@@ -40,47 +48,74 @@
       color: $color-primary-cerise;
     }
 
-    @media screen and (min-width: 576px) {
+    @media (min-width: 576px) {
       flex-direction: column-reverse;
 
+      @media (max-width: 991px) {
+        padding-right: 3em;
+      }
+
       span {
         align-self: flex-end;
       }
     }
+
+    @media (max-width: 575px) {
```

Comment:
I've changed the others as they are bootstrap sizes, but this one is used only here, so I don't see a reason for a variable